styles/layout.css

```diff
@@ -1,14 +1,27 @@
-.template {
-  display: none;
+#articles {
   text-align: center;
-  margin: 0 auto;
 }
 
-a {
```

Comment:
Since ids are unique and CSS is all about reusing the same rules, it's considered best practice to try and generalize your CSS by using classes. ids are good for when you need to attach JS hooks to something, but they lead to a lot of duplicate CSS.

That said, if you do have some sort of container where you want everything in there to be completely uniquely styled, you can and should use an id there. However, that situation is very rare.